src/_pytest/assertion/rewrite.py

```diff
@@ -684,12 +684,9 @@ def run(self, mod: ast.Module) -> None:
         if not mod.body:
             # Nothing to do.
             return
+
         # Insert some special imports at the top of the module but after any
         # docstrings and __future__ imports.
```

Comment:
this comment has been orphaned

Reply:
That comment is still relevant in its current place! L690-L720 is figuring out the "after any docstrings and __future__ imports part".

Reply:
it's 40 lines away from the code that's relevant to the comment (the "special imports")

Reply:
Okay, made a change! :-)